### ISSUE TITLE
Поддержка

### ISSUE THREAD

**Issue by nelegal4eg** [QtExcel/QXlsx#315]
Приветствую. Проект еще поддерживается?

**Comment by Jihadist** [comment]
@nelegal4eg да, основные мейнтейнеры указаны тут https://github.com/orgs/QtExcel/people
